db/migrate/20170831092724_create_refunds.rb

```diff
@@ -0,0 +1,74 @@
+class CreateRefunds < ActiveRecord::Migration
+  def change
+    create_table :refunds do |t|
```

Comment:
That's a big table.

Reply:
It is !!  Intentionally kept nice and flat for ease of exporting and so that associated addresses etc.. do not get mixed in with normal ET data which would have forced us to do things like polymorphic associations etc...

Reply:
Since these changes are permanent, I would suggest to not approach this as a quick solution. I think we should go through the new data we are going to introduce into the system and see if it makes sense to re-think their structure in the db.

@garytaylor is it possible to give us an idea of the new resources we are introducing with this feature?

Reply:
The thing is, we are collecting information and not doing anything with it apart from exporting it, so converting into a relational structure for all the addresses, respondents etc... then converting it back again to export it seems completely pointless.

If we were doing anything with the data, then yes - we would do this a different way.

What is your motivation for wanting to do this differently ?